### PR TITLE
Ch api testing #108929430

### PR DESCRIPTION
**Pull request by andela-bebowe**

### CODE REVIEW

**Comment by andela-bebowe** [comment]
This failure has been fixed in Oscar's branch.

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -46,7 +49,7 @@
   end
   config.infer_spec_type_from_file_location!
   config.include ApplicationHelper
-  config.include Requests::JsonHelpers, type: :controller
+  config.include Requests::JsonHelper, type: :controller
```
Comment (andela-iokonkwo) [inline spec/rails_helper.rb]: Can you clarify why this line is important?

Comment (andela-bebowe) [inline spec/rails_helper.rb]: I am using it to add methods from the Request Helper file to my controller.
Did not want to do it in each file

Comment (andela-bebowe) [inline spec/rails_helper.rb]: Some methods include the json method that takes does JSON.parse(response.body)

Comment (andela-iokonkwo) [inline spec/rails_helper.rb]: Basically you want to call `json` as against `JSON.parse`

**Comment by andela-iokonkwo** [comment]
Your branch name do not follow our branch naming convention. Please remove the `#` in the name.

File: spec/rails_helper.rb (hunk 1 above)
Comment (andela-bebowe) [inline spec/rails_helper.rb]: Yes

**Pull request closed without merging**